internal/mesh/internal/controllers/routes/routestest/routestest.go

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
```

Comment:
This package is meant to be helper code to facilitate turning a pile of resources into a `ComputedRoutes` resource without requiring the author of a test to manually populate the fields.